rocket-chat-android-widgets/src/test/kotlin/UserAvatarHelperTest.kt

```diff
@@ -0,0 +1,22 @@
+import chat.rocket.android.widget.helper.UserAvatarHelper
```

Comment:
Tests should live in the same package (and directory structure) as the class you are testing...